<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Party List</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(58, 58, 58);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="table_party_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58);
color: rgb(250, 250, 250);
</string>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>161</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" stretch="1,0,0,0,0,0">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="button_new">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: rgb(250, 250, 250);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(69, 86, 206), stop:1 rgb(69, 206, 86));
border-style: solid;
border-radius:10px;
padding-left: 5px; 
padding-right: 5px;
padding-top: 5px; 
padding-bottom: 5px;</string>
       </property>
       <property name="text">
        <string>Add new ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: rgb(250, 250, 250);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(69, 86, 206), stop:1 rgb(69, 206, 86));
border-style: solid;
border-radius:10px;
padding-left: 5px; 
padding-right: 5px;
padding-top: 5px; 
padding-bottom: 5px;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete_all">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: rgb(250, 250, 250);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(69, 86, 206), stop:1 rgb(69, 206, 86));
border-style: solid;
border-radius:10px;
padding-left: 5px; 
padding-right: 5px;
padding-top: 5px; 
padding-bottom: 5px;</string>
       </property>
       <property name="text">
        <string>Delete all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete_all_2">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: rgb(250, 250, 250);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(69, 86, 206), stop:1 rgb(69, 206, 86));
border-style: solid;
border-radius:10px;
padding-left: 5px; 
padding-right: 5px;
padding-top: 5px; 
padding-bottom: 5px;</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Microsoft YaHei UI&quot;;
font-weight: bold;
color: rgb(250, 250, 250);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(69, 86, 206), stop:1 rgb(69, 206, 86));
border-style: solid;
border-radius:10px;
padding-left: 5px; 
padding-right: 5px;
padding-top: 5px; 
padding-bottom: 5px;</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>button_new</zorder>
    <zorder>button_delete</zorder>
    <zorder>button_delete_all</zorder>
    <zorder>button_close</zorder>
    <zorder>button_delete_all_2</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
